compiler/src/iree/compiler/Codegen/LLVMCPU/test/materialize_x86_64_launch_configuration.mlir

```diff
@@ -1225,7 +1225,7 @@ hal.executable private @matmul_odd {
     }
   }
 }
-//  CHECK-DAG: #[[CONFIG:.+]] = #iree_codegen.lowering_config<tile_sizes = {{\[}}[11, 32, 0], [8, 32, 0], [0, 0, 16], [0, 0, 0]]>
+//  CHECK-DAG: #[[CONFIG:.+]] = #iree_codegen.lowering_config<tile_sizes = {{\[}}[11, 25, 0], [8, 25, 0], [0, 0, 16], [0, 0, 0]]>
```

Comment:
What are we trying to fix with this PR? 

The problem I see for this case is that the distribution tile size goes from 32 -> 25, which is not a multiple of the vector size. Vector-wise, we would process then 32 elements with mask or 16 (main vector loop) + 9 (peeled loop), which is worse than the original configuration.

Reply:
This is for distribution, not vector level things. It is 25 because the dimension size is `49`. If there are two threads and we want to divide the work evenly, shouldn't it be 25 instead of 32?

The vector size is 25 because we used `8, 32, 16` for matmul. Setting tile size to `32` or `25` means the same thing for this shape of matmul.